src/coreclr/src/inc/sospriv.idl

```diff
@@ -390,3 +390,14 @@ interface ISOSDacInterface7 : IUnknown
     HRESULT GetProfilerModifiedILInformation(CLRDATA_ADDRESS methodDesc, struct DacpProfilerILData *pILData);
     HRESULT GetMethodsWithProfilerModifiedIL(CLRDATA_ADDRESS mod, CLRDATA_ADDRESS *methodDescs, int cMethodDescs, int *pcMethodDescs);
 };
+
+[
+    object,
+    local,
+    uuid(3f5f88ae-a312-42c1-affa-ecedfaf248c0)
+]
+interface ISOSDacInterface8 : IUnknown
+{
+    HRESULT GetAssemblyLoaderAllocatorData(CLRDATA_ADDRESS loaderAllocator, struct DacpAssemblyLoaderAllocatorData* pAssemblyLoaderAllocatorData);
```

Comment:
Where do you get the address of the loader allocator or the binder from? Do we also need APIs to e.g. get loader allocator from MethodTable or MethodDesc?

Reply:
We already have that, I have added that quite some time ago.
```
interface ISOSDacInterface6 : IUnknown
{
    HRESULT GetMethodTableCollectibleData(CLRDATA_ADDRESS mt, struct DacpMethodTableCollectibleData *data);
};
```
I also have the SOS change tested and ready to get in.

Reply:
`GetMethodTableCollectibleData` returns `LoaderAllocatorObjectHandle`. How do you get from `LoaderAllocatorObjectHandle` to `loaderAllocator` ?

>  SOS change

It may be useful to add link to the SOS change for context.

Reply:
See https://github.com/dotnet/runtime/issues/11157#issuecomment-642288660 for the steps to get from an object to the AssemblyLoadContext. There were two steps that couldn't be done via DAC before, those are the ones I'm adding here.

As for the SOS change, I was planning to create the PR after this one gets in.

Reply:
You may want to write some of the SOS code to test these new DAC APIs to test the binder code added to the DAC has been properly "DACized" (dac_cast<T> macros, etc).